src/scss/components/layout/_index.scss

```diff
@@ -1,9 +1,9 @@
-@import (reference) 'vars';
-@import (reference) 'components/box/index';
+@import 'vars';
+@import 'components/box/index';
```

Comment:
It looks like the Layout component may not need anything from the Box component.

Can you remove the remove the import to the Box component (line 2)?

Reply:
please check the changes in latest commit